File: Nível 2/2311 - Saltos Ornamentais/main.lua
quantia = tonumber(io.read())

for i = 1, quantia do
    nomeAtleta = io.read()
    dificuldade = tonumber(io.read())
    notas = {}
    for num in string.gmatch(io.read(), "%S+") do
        table.insert(notas, tonumber(num))
    end

    max_value = math.max(table.unpack(notas))
    for j, num in ipairs(notas) do
        if num == max_value then
            table.remove(notas, j)
            break
        end
    end

    min_value = math.min(table.unpack(notas))
    for j, num in ipairs(notas) do
        if num == min_value then
            table.remove(notas, j)
            break
        end
    end

    soma = 0
    for j, num in ipairs(notas) do
        soma = soma + num
    end

    notaFinal = soma * dificuldade
    print(string.format("%s %.2f", nomeAtleta, notaFinal))
end

File: Nível 2/2311 - Saltos Ornamentais/main.py
quantia = int(input("A: "))

vencedores = []

for i in range(1, quantia + 1):
    nomeAtleta = input("B: ")
    dificuldade = float(input("C: "))
    notas = list(map(float, input("D: ").split(" ")))
    notas.remove(max(notas))
    notas.remove(min(notas))
    soma = sum(notas)
    notaFinal = soma * dificuldade
    vencedores.append(f"{nomeAtleta} {notaFinal:.2f}")

for valor in vencedores:
    print(valor)